images/ipfailover/keepalived/lib/config-generators.sh

```diff
@@ -178,7 +178,7 @@ function generate_vip_section() {
 #
 function generate_vrrpd_instance_config() {
   local servicename=$1
-  local iid=${2:-"0"}
+  local iid=${2:-"1"}
```

Comment:
Any reason to set it to 1 instead - that changes the value of virtual router id below to always start from 1 - 
`virtual_router_id $((vrrpidoffset + iid))`